src/api/variables.ts

```diff
@@ -51,9 +51,8 @@ export class Variables extends Map<string, string> {
   toPaseVariables(): Record<string, string> {
     const variables: Record<string, string> = {};
     for (const [key, value] of this.entries()) {
-      //Env variables can't contain * or {}; keeping only &xxx variables
-      if(key.startsWith('&')){
-        variables[key.substring(1)] = value;
+      if ((/^[A-Za-z\&]/i).test(key)) {
+        variables[key.startsWith('&') ? key.substring(1) : key] = value;
       }
```

Comment:
To be thorough, we should test the actual [naming rule for shell variable](https://www.gnu.org/software/bash/manual/html_node/Definitions.html#index-name), stripping `&` beforhand.
```suggestion
      const cleanKey = key.startsWith('&') ? key.substring(1) : key;
      if ((/^[a-z_]\w*$/i).test(cleanKey)) {
        variables[cleanKey] = value;
      }
```